src/main/java/gift/domain/user/UserRestController.java

```diff
@@ -41,4 +52,36 @@ public ResponseEntity<SimpleResultResponseDto> login(@Valid @RequestBody UserDTO
         return ResponseMaker.createSimpleResponseWithJwtOnHeader(HttpStatus.OK, "로그인에 성공했습니다", jwt);
     }
 
+    /**
+     * 카카오 회원 로그인
+     */
+    @GetMapping("/kakaoLogin")
```

Comment:
RESTful한 API 규칙에서는 다음과 같이 작성하는 것을 [권장](https://blog.restcase.com/5-basic-rest-api-design-guidelines/)합니다! 대소문자를 활용할 경우 혼란이 가중되어서요ㅎㅎ
```suggestion
    @GetMapping("/kakao-login")
```

Reply:
앞으로 많이 사용될 것 같습니다. 꿀팁 정말 감사합니다!!